nrf_modem/doc/flavour_features.rst

```diff
@@ -0,0 +1,42 @@
+.. _flavour_features:
+
+Flavours and features
+#####################
+
+The Modem library comes in two variants, or flavours, each with a specific feature set and support for a specific variant of modem firmware.
```

Comment:
Maybe internal, though dont we use variant for dect_phy/cellular/... and flavour for standard/with logging/...?